main/src/library/LazyList.flix

```diff
@@ -420,7 +414,7 @@ namespace LazyList {
     ///
     /// The function `f` must be pure.
     ///
-    pub def exists(f: a -> Bool & ef, l: LazyList[a]): Bool & ef = match l {
+    pub def exists(f: a -> Bool, l: LazyList[a]): Bool = match l {
```

Comment:
I think these should be allowed to be effect polymorphic.

Reply:
Ah yeah I see there were some merge conflicts. They should definitely be effect polymorphic.